_docs/05-external-guides.md

```diff
@@ -24,13 +24,21 @@ Information to assist with building assets for Tabletop Simulator.
 | [Universal Blackjack Reboot Modding Guide](https://steamcommunity.com/sharedfiles/filedetails/?id=919849162) | This guide is simply a list of how you can enact changes on certain elements of the table. |
 | [Creating Decks and Sheets](https://steamcommunity.com/sharedfiles/filedetails/?id=1349263773) | Brief guide to everything decks and sheets with the aim to educate on how tiling, sheets and the deck importer works and how to achieve high quality and optimized results. |
 
-## Scripting
+## Lua Scripting
 
 | Guide | Description |
 | --- | --- |
 | [A beginner's guide to scripting](https://steamcommunity.com/sharedfiles/filedetails/?id=714904631) | Learn the basics of how lua works and some of the basic building blocks of programming. |
 | [An intermediate guide to scripting](https://steamcommunity.com/sharedfiles/filedetails/?id=879449506) | This guide is designed to teach you some common uses for Lua. It includes multiple examples of how to use certain functions and common concepts in Tabletop Simulator. |
 | [Useful functions for scripting](https://steamcommunity.com/sharedfiles/filedetails/?id=752690530) | A list of various functions I have written or accrued in order to perform certain actions. Designed to be copy-pasted directly into your code for ease of use. |
+| [Toggle Switches](https://steamcommunity.com/sharedfiles/filedetails/?id=1432389368) | A basic guide on how to create simple toggle switches with Lua. |
+
+## XML Scripting
```

Comment:
Nitpick, but I'd prefer just "XML", as technically XML is not scripting.

Reply:
Ok